.circleci/bin/test-airflow-image.py

```diff
@@ -216,12 +216,18 @@ def test_apache_airflow_in_requirements(tmp_path):
     test_project.mkdir()
     image_name = get_image_name(ImageType.ONBUILD.value)
 
+    restricted_value_patterns = [
+        "apache-airflow==1.10.10", "apache-airflow>=1.10.5", "apache-airflow~=1.10.7"
```

Comment:
Try with spaces around the operator too?

Reply:
Done in https://github.com/astronomer/ap-airflow/pull/124/commits/a2ae1b6794968818bd989a2e438cc0850b0c4ca4